update ```aiogram``` and ```aiogram-dialog```

vsecoder opened the pull request:
aiogram release 3.0.0
aiogram-dialog release 2.0.0

vsecoder committed:
update ```aiogram``` and ```aiogram-dialog```

aiogram release 3.0.0
aiogram-dialog release 2.0.0

Codwizer commented:
Cool

blind0 commented on README.md:
```diff
@@ -12,10 +12,10 @@
 * ![pyrogram](https://img.shields.io/badge/latest-pyrogram-orange) (Optional) for MTProto requests, such as bulk delete,
   resolve by username and list participants in a group
-* ![aiogram-dialog](https://img.shields.io/badge/beta--2.x-aiogram__dialog-green) (Optional) for creating multi-step
+* ![aiogram-dialog](https://img.shields.io/badge/2.0.0-aiogram__dialog-green) (Optional) for creating multi-step
```
```suggestion
* ![aiogram-dialog](https://img.shields.io/badge/2.2.0-aiogram__dialog-green) (Optional) for creating multi-step
```
newer version

blind0 commented on app/__main__.py:
```diff
@@ -95,7 +91,11 @@ async def main():
+        )
+    )
     token = config.bot.token
     bot_settings = {"session": session, "parse_mode": "HTML"}
```
```suggestion
    bot_settings = {"session": session}
```
deprecated since aiogram=3.7.0

blind0 commented on requirements.txt:
```diff
@@ -1,6 +1,6 @@
 aerich
-aiogram-dialog==2.0.0b11
-aiogram==3.0.0b5
+aiogram-dialog==2.0.0
```
```suggestion
aiogram-dialog==2.2.0
```
newer version aiogram-dialog

blind0 commented on requirements.txt:
```diff
@@ -1,6 +1,6 @@
-aiogram-dialog==2.0.0b11
-aiogram==3.0.0b5
+aiogram-dialog==2.0.0
+aiogram==3.0.0
```
```suggestion
aiogram==3.*
```
latest aiogram 3 version

blind0 commented on app/dialogs/__init__.py:
we can do a router parser
```python
from aiogram import Router
import os
from importlib import import_module


def get_dialog_router() -> Router:
    router = Router()
    for module in os.listdir(os.path.dirname(__file__)):
        if module == "__init__.py" or module[-3:] != ".py":
            continue
        router.include_router(getattr(import_module(f".{module[:-3]}", __package__), "ui"))
    return router
```

blind0 reviewed (changes requested):
update pls

vsecoder committed:
updates by @blind0

- latest aiogram version
- dialogs router parser
- update aiogram dialogs version

vsecoder committed:
fix deps and deepsource

- use DefaultBotProperties
- `run_in_transaction` on 0.20.0 tortoise
- fixes by deepsource

vsecoder committed:
update req to last aiogram (3.13.1)
